src/ggrc/migrations/versions/20200401_8b0d2c4e1c4b_remove_processes_from_systems_table.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2020 Google Inc.
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+
+"""
+remove processes from systems table
+
+Create Date: 2020-04-01 11:17:05.144312
+"""
+# disable Invalid constant name pylint warning for mandatory Alembic variables.
+# pylint: disable=invalid-name
+
+import sqlalchemy as sa
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = '8b0d2c4e1c4b'
+down_revision = '10bcbe93e3b7'
+
+
+GET_PROCESS_IDS = """
+    SELECT id FROM systems
+    WHERE is_biz_process = 1
+"""
+
+DELETE_PROCESSES_BY_IDS = """
+    DELETE FROM systems
+    WHERE id IN :process_ids
+"""
```

Comment:
There is no need to select IDs to delete processes.
```suggestion
DELETE_PROCESSES = """
    DELETE FROM systems
    WHERE is_biz_process = 1
"""
```

Reply:
Done